pkgs/top-level/all-packages.nix

```diff
@@ -5214,7 +5214,7 @@ with pkgs;
   firestarter = callPackage ../applications/misc/firestarter { };
 
   fits-cloudctl = callPackage ../tools/admin/fits-cloudctl { };
-
+  
```

Comment:
This change triggers the failure in the .editorconfig check as it introduces a trailing whitespace.